crates/utils/src/utils/validation.rs

```diff
@@ -238,54 +237,6 @@ pub fn clean_url_params(url: &Url) -> Url {
   url_out
 }
 
-pub fn check_totp_2fa_valid(
```

Comment:
Why move this out of validation?

Reply:
No need to make it available everywhere when its only used in a single crate. Also iirc there was some dependency problem with the old location but I dont remember the details now.